### ISSUE TITLE
questions about PPT

### ISSUE THREAD

**Issue by jdjiang312** [Pointcept/Pointcept#418]
Hi, your work on Point Prompt Training is really intriguing. We have some questions about PPT：
1. We're currently trying to replicate the PPT work on our own dataset (which is about 100GB after preprocessing). However, due to limited computational resources, we're struggling to balance the **grid size** and **batch size**. For instance, when using Sp U-Net as the backbone and training on 4 GPUs (with each 32GB RAM), **if we set the grid size to 0.04, the batch size can only be set to 4 (1 batch per GPU)**. On the other hand, **if we set the grid size to 0.1, the batch size can be increased to 8 (2 batches per GPU)**. Could you give us some advice on **whether we should aim for a larger batch size or a larger grid size to make PPT effective**?

2. When we use PPT, **the mIoU isn't as good as when we train on a single dataset.** Is this normal for pre-training? Do we need to fine-tune the pre-trained model on a specific dataset to achieve the accuracy described in your paper?

I hope you can give us some advices. Thanks in advance.
